justspaces/settings.py

```diff
@@ -151,4 +151,4 @@
 
 FOBI_DEFAULT_THEME = 'foundation5'
 FOBI_THEME_FOOTER_TEXT = gettext.gettext('')
-FOBI_RESTRICT_PLUGIN_ACCESS = False
+FOBI_RESTRICT_PLUGIN_ACCESS = True
```

Comment:
I'm not totally confident I get what this is doing -- from the docs:

> FOBI_RESTRICT_PLUGIN_ACCESS (bool): If set to True, (Django) permission system for dash plugins is enabled. Defaults to True. Setting this to False makes all plugins available for all users.

Does this mean that some users won't have access to some plugins, or that plugins can only be used on the group of which they're a part?

Reply:
I changed this when I was trying to figure out how to approach this issue and forgot to change it back—my bad! I think we shouldn't use Fobi's permissions system but should instead build our own once we get there.